site2/docs/io-cdc-debezium.md

```diff
@@ -181,7 +181,7 @@ You need to create a configuration file before using the Pulsar Debezium connect
 
 ### Configuration
 
-You can use one of the following methods to create a configuration file.
+You can use one of the following methods to create a configuration file. *decoderbufs* is the default logical decoding output plug-in in Pulsar Debezium connector, you can change the the logical decoding by parameter *plugin.name* ,more about *plugin.name* information click [here](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html)
```

Comment:
```suggestion
You can use one of the following methods to create a configuration file. *decoderbufs* is the default logical decoding output plug-in in Pulsar Debezium connector, you can change the the logical decoding using the *plugin.name* parameter. For more details about *plugin.name*, see [postgresql guide](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html)
```

Reply:
PostgreSQL, not postgresql

Reply:
@88070173 Here needs to update to "PostgreSQL" as well.

when you modify some contents, please check all occurrences and keep them consistent. In this way, it can save reviewers' time and effort to check PRs again and again.